pkg/helm/helm.go

```diff
@@ -308,14 +347,31 @@ func (h *Helm) LoadChart(ref *LoadChartConfig) (c *chart.Chart, err error) {
 	if err != nil {
 		return
 	}
+
 	log.Printf("Using chart path %v", chartPath)
+
 	// Check chart requirements to make sure all dependencies are present in /charts
-	if c, err = chartutil.Load(chartPath); err != nil {
+	if c, err = loadChartWithLockfile(chartPath, ref.LockFile); err != nil {
```

Comment:
Do you also think it makes sense to enforce that the lock file can be loaded from a sub directory only (using `securePath`)? (While I didn't properly do it for the chart path I did it for the values files already)